build.gradle

```diff
@@ -177,7 +177,8 @@ dependencies {
     implementation "software.amazon.cryptography:aws-cryptographic-material-providers:1.9.0"
     implementation "org.dafny:DafnyRuntime:4.10.0"
     implementation "software.amazon.smithy.dafny:conversion:0.1.1"
-    implementation "org.bouncycastle:bcprov-jdk18on:${versions.bouncycastle}"
+    // needed by aws-encryption-sdk-java
+    implementation "org.bouncycastle:bc-fips:2.0.0"
```

Comment:
Switch this to `implementation "org.bouncycastle:bc-fips:${versions.bouncycastle_jce}"` when FF updates to beta1 qualifier.

This is added in the [version catalog](https://github.com/opensearch-project/OpenSearch/blob/main/gradle/libs.versions.toml#L63) after the cut to beta1 and is not available in the alpha1 artifacts.